daft/series.py

```diff
@@ -63,18 +54,7 @@ def from_arrow(array: pa.Array | pa.ChunkedArray, name: str = "arrow_series") ->
             return Series.from_pylist(array.to_pylist(), name=name, pyobj="force")
         elif isinstance(array, pa.Array):
             array = ensure_array(array)
-            if _RAY_DATA_EXTENSIONS_AVAILABLE and isinstance(array.type, ArrowTensorType):
-                storage_series = Series.from_arrow(array.storage, name=name)
-                series = storage_series.cast(
-                    DataType.fixed_size_list(
```

Comment:
Don't we need to port this logic into our data conversions from Ray datasets?

Reply:
Ah on the ray runner side I didn't see any conversion going through `Series.from_arrow`, but I see now that it's at the very bottom of the stack after calling `Micropartition.from_arrow`.

Reply:
(Bad tests!)

Reply:
@jaychia added the missing logic, PTAL!

Reply:
Will also try to add a test around this (ᵕ—ᴗ—)